company_website/templates/estimate_project/estimate_project_form.haml

```diff
@@ -63,7 +63,7 @@
               I have read and accepted our
               %a{:href => "{% url 'privacy_and_policy' %}"}  Privacy Policy
             {% url "thank_you" as thank_you_url %}
-            %input.custom-button{:type => "submit", :value => "Estimate Project", :class => "estimate-button", :id => "btnSubmit", :onclick => "return gtag_report_conversion('{{ thank_you_url }}')"}
+            %input.custom-button{:type => "submit", :value => "Estimate Project", :class => "estimate-button", :id => "btnSubmit", :onclick => 'return gtag_report_conversion("https://{{ request.get_host }}{{ thank_you_url }}")'}
```

Comment:
`https` - It won't be always with ssl because locally and on test server we don't have any cert. We have such variable in our settings like `URL_PREFIX`. Please use this variable to differentiate correct prefix

Reply:
It doesn't really matter anyway, because it will only work on the production, or so we're told.
Guess I could've just as well hardcoded **https://codepoets** 🤷

Reply:
Done, I've passed the URL_PREFIX to the context data.